tests/functional/spec/components/checkbox/mutually_exclusive/mutually_exclusive_multiple_day_month_year_date.spec.js

```diff
@@ -2,206 +2,212 @@ import DatePage from "../../../../generated_pages/mutually_exclusive_multiple/mu
 import SummaryPage from "../../../../generated_pages/mutually_exclusive_multiple/mutually-exclusive-date-section-summary.page";
 
 describe("Component: Mutually Exclusive Day Month Year Date With Multiple Radio Override", () => {
-  beforeEach(() => {
-    browser.openQuestionnaire("test_mutually_exclusive_multiple.json");
-    browser.url("/questionnaire/mutually-exclusive-date");
```

Comment:
Why do we remove the browser.url() here?

Reply:
I had a look. It looks the questionnaire never fully loads in time so the browser.url() would not load either. If we add an anything between like `await browser.scroll(0,200)`, it works again. Is there any preference to what we should have between.

Reply:
I personally prefer adding the `await browser.scroll(0,200)`. That way we make it explicit to the reader what the issue is. Otherwise, we might be confused as to why we duplicate the `browser.url` in each test.

But it's no biggie, happy to hear what other people think

Reply:
Is `scroll` important, if not, I suggest a `browser.pause()` or a `waitUntil()`?. Thoughts?

Reply:
Scroll is not important we can use `browser.pause()`. I do not have any issues with either pause or scroll

Reply:
I think we should use `browser.pause()` here. It's possible that `browser.scroll()` works here because of the time it gives the page to finish loading whilst scrolling. Not necessarily because the element cannot be found. In which case we might as well use `browser.pause()` which is designed to wait for the page to load fully.

Reply:
Happy with `pause()` and the changes